extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -57,7 +57,9 @@ export function registerAzureResourceCommands(appContext: AppContext, azureViewT
 					azureAccount = accounts.find(acct => acct.displayInfo.displayName === pickedAccount);
 				}
 			}
-
+			if (!azureAccount) {
+				throw new Error('No Azure Account chosen');
```

Comment:
this seems something that needs to be localized

Reply:
Personally I'm against localizing error messages like this- we can't control any libraries we use and the errors they may throw and so it's better to instead catch errors and then display a localized error message with additional information as needed. And we want error messages in console logs to not be localized so that when we ask customers for logs we don't have to translate them ourselves. 

We should have a discussion about this though, I'll set something up for that. Given that most of the rest of azurecore also doesn't localize errors at the moment though I'm going to leave these unlocalized until we come to a decision.